src/TransportStream.ts

```diff
@@ -1,14 +1,16 @@
 import { EventEmitter } from 'events';
 import { TelnetStream } from '../types/TelnetStream';
+import { TelnetSocket } from '../types/TelnetSocket';
+import { RanvierWebSocket } from '../types/RanvierWebSocket';
 import { WebsocketStream } from '../types/WebsocketStream';
 
-export type SocketType = TelnetStream | WebsocketStream;
-
+export type StreamType = TelnetStream | WebsocketStream;
```

Comment:
I had to work around this in my repo after making a socket.io stream. Not a big deal though and it wasn't hard to finagle